app/locale/en.coffee

```diff
@@ -1291,6 +1291,7 @@
     edit_settings1: "Edit Class Settings"
     add_students: "Add Students"
     stats: "Statistics"
+    student_email_invite_blurb: "Your students can also use class code <strong>__classCode__</strong> when creating a Student Account, no email required."
```

Comment:
This seems a bit dangerous to me. Better would be to have this be in markdown, and then use `marked` with `sanitize:true`. I suppose it's a long shot that someone would submit a translation with a nefarious script and we wouldn't notice, but still...

Reply:
Where can I find a markdown example?